src/cmd/file.go

```diff
@@ -0,0 +1,131 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/manifoldco/promptui"
+	"github.com/radiusmethod/kxd/src/utils"
+	"github.com/spf13/cobra"
+	"log"
+	"os"
+	"path/filepath"
+	"sort"
+	"strings"
+)
+
+var fileCmd = &cobra.Command{
+	Use:     "file",
+	Short:   "Kubeconfig file command",
+	Aliases: []string{"f"},
+	Long:    "This is the default file command.",
+	//	Should this default to file switch?
```

Comment:
should config switcher be run if `kxd file` ran without `kxd file switch`?

Reply:
i would keep it explicit for now